csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -23,13 +23,14 @@
     <!-- Include PDB in the built .nupkg -->
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <IsTrimmable>true</IsTrimmable>
+    <EnableAOTAnalyzer>true</EnableAOTAnalyzer>
```

Comment:
Is this available with the .NET 6 SDK, if we *do* stick with that?

Reply:
It's new in .NET 7 SDK - https://github.com/dotnet/sdk/pull/24091

Reply:
Ah, that's interesting. So presumably the project *could* be built with the .NET 6 SDK and the property would be ignored, but with .NET 7 it can sniff out problematic bits? Hmm.